modules/openy_gc_personal_training/openy_gc_personal_training.module

```diff
@@ -143,27 +144,6 @@ function openy_gc_personal_training_personal_training_update(PersonalTrainingInt
     ->entityUpdate($personal_training);
 }
 
-/**
- * Implements hook_ENTITY_TYPE_delete() for personal_training entities.
- */
-function openy_gc_personal_training_personal_training_delete(PersonalTrainingInterface $personal_training) {
-  if ($personal_training->bundle() === 'personal_training') {
-    \Drupal::service('class_resolver')
-      ->getInstanceFromDefinition(PersonalTrainingItemsOperations::class)
-      ->entityDelete($personal_training);
-
-    \Drupal::service('class_resolver')
-      ->getInstanceFromDefinition(VyFavoriteItemsManager::class)
-      ->deleteRelatedFavoriteItems($personal_training);
-
-  }
-  elseif ($personal_training->bundle() === 'training_series') {
-    \Drupal::service('class_resolver')
-      ->getInstanceFromDefinition(PersonalTrainingSeriesOperations::class)
-      ->entityDelete($personal_training);
-  }
-}
-
 /**
  * Implements hook_theme().
```

Comment:
```suggestion
 * Implements hook_mail().
```

Reply:
Isn't this doc block before `openy_gc_personal_training_theme` function?